spec/optimizely_factory_spec.rb

```diff
@@ -57,98 +72,98 @@ class CustomConfigManager
 
       custom_config_manager = CustomConfigManager.new
       optimizely_instance = Optimizely::OptimizelyFactory.default_instance_with_config_manager(custom_config_manager)
-      expect(optimizely_instance.config_manager). to eq(custom_config_manager)
+      expect(optimizely_instance.config_manager). to be(custom_config_manager)
     end
   end
 
   describe '.custom_instance' do
-    it 'should take http config manager when sdk key is given' do
-      allow(Optimizely::HTTPProjectConfigManager).to receive(:new)
-      http_project_config_manager = Optimizely::HTTPProjectConfigManager.new(
-        sdk_key: 'sdk_key',
-        datafile: datafile,
-        logger: spy_logger,
-        error_handler: error_handler,
-        skip_json_validation: false,
-        notification_center: notification_center
-      )
-
+    it 'should take http config manager when sdk key, polling interval, blocking timeout are given' do
+      Optimizely::OptimizelyFactory.set_polling_interval(50)
```

Comment:
Do we really need this case, it's already defined. 
https://github.com/optimizely/ruby-sdk/pull/204/files#diff-a0f73069c4d39dc48b54a5beba886129R50

Reply:
Yes, this checks custom_instance convenience method. While the other one verifies default_instance method